docs/examples.rst

```diff
@@ -0,0 +1,178 @@
+.. doctest-skip-all
+
+Examples
+========
+Note that these examples are not tested by continuous integration tests; it is
```

Comment:
@keflavich - just out of curiosity, why not? Can't we put the cube on data.astropy.org?

Reply:
Maybe?  I don't have any experience doing that.  Let's make CI-testing the examples an Issue.